<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphSettings3dDialog</class>
 <widget class="QDialog" name="GraphSettings3dDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia okna wykresu 3d</string>
  </property>
  <property name="styleSheet">
   <string notr="true">ColorButton {text-align:left; padding-left:10px}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="showEdges">
       <property name="text">
        <string>Pokaż krawędzie kreślenia</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorButton" name="edgesColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor krawędzi kreślenia</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="showLabels">
       <property name="text">
        <string>Pokaż etykiety</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorButton" name="labelsColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor etykiet</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ColorButton" name="bgColorBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Kolor tła</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Kierunek rotacji wykresu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #777</string>
        </property>
        <property name="text">
         <string>(z perspektywy dolnego odcinka osi Z)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rotateCW">
        <property name="text">
         <string>zgodnie z ruchem wskazówek zegara</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rotateACW">
        <property name="text">
         <string>odwrotnie do ruchu wskazówek zegara</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="drawInInterval">
     <property name="title">
      <string>Kreśl w przedziale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max. X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Min. X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="intervalXFrom"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="intervalYFrom"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Min. Y:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Min. Z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Max. Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="intervalXTo"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="intervalYTo"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Max. Z:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="intervalZFrom"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="intervalZTo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showCSMiniature">
     <property name="text">
      <string>Pokaż miniaturę układu współrzędnych</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showDirectionsMarkers">
     <property name="text">
      <string>Pokaż oznaczenia kierunków</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showLegend">
     <property name="text">
      <string>Pokaż legendę</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphSettings3dDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphSettings3dDialog</receiver>
   <slot>checkSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkSettings()</slot>
 </slots>
</ui>
